services/frontend-service/src/ui/components/button/button.tsx

```diff
@@ -39,7 +39,10 @@ export const Button = (props: { className?: string; label?: string; icon?: strin
             <div className="mdc-button__ripple" />
             {!!icon && (
```

Comment:
this is two levels of complicated. you can remove the `<i/>` element and maybe one of the checks 

(if needed, add the classnames to the icon JSX)